PoroElastic/SIMPoroElasticity.h

```diff
@@ -66,7 +66,7 @@ template<class Dim> class SIMPoroElasticity : public SIMElasticityWrap<Dim>
   }
 
   //! \brief Evaluates some iteration norms for convergence assessment.
-  //! \param[in] x Global primary solution vector
+  //! \param[in] u Global primary solution vector
```

Comment:
A sin of mine, it seems. Can squash this into your first commit since we don't allow rewriting the history on master

Reply:
Done :+1: